hyrax/config/fedora.yml

```diff
@@ -1,12 +1,12 @@
 development:
   user: fedoraAdmin
   password: fedoraAdmin
-  url: <%= ENV['FEDORA_URL_SCHEME'] || 'http' %>://<%= ENV['FEDORA_HOST'] || 'localhost' %>:<%= ENV['FCREPO_DEVELOPMENT_PORT'] || 8984 %>/rest
+  url: <%= ENV['FEDORA_URL_SCHEME'] || 'http' %>://<%= ENV['FEDORA_HOST'] || 'localhost' %>:<%= ENV['FEDORA_PORT'] || 8080 %>/fcrepo/rest
```

Comment:
I'm not sure we do want all of the envs to use the same default port - running this on a local machine can be a pain with collisions on 8080 - quite a popular port.

In any case, I've changed the config wrapper to bring it in line with your change here to fix this PR.